subjects/atm-management-system/audit/README.md

```diff
@@ -82,13 +82,13 @@
 
 #####   Open two terminals and login with two different users. Then transfer ownership of an account to the other user.
 
-###### +Was the user whom received the account notified instantly?
+###### +Was the user who received the account notified instantly?
 
 ###### +Did the student update the terminal interface?
 
 ###### +Is the password saved in the file `"users.txt"` encrypted?
 
-###### +Did the student create a relational database? • 6
+###### +Did the student create a relational database?
```

Comment:
The "• 6" must not be removed.
It is part of the structure of the audit, it is a way to give weight to the bonus questions so that we can give more or less XP points